src/Components/Header/index.js

```diff
@@ -4,14 +4,17 @@ import ProfileMenu from './ProfileMenu';
 function Header() {
   const token = sessionStorage.getItem('token');
   const logo = `${process.env.PUBLIC_URL}/assets/images/logo.png`;
+  const linkUrl = `http://localhost:3000/auth/home`;
   return (
     <header>
       <div className={styles.container}>
         <div className={styles.containerLogo}>
-          <div className={styles.imgLogo}>
-            <img src={logo} alt="Rocket logo" />
-          </div>
-          <div className={styles.brand}>Mega Rocket Gym</div>
+          <a href={linkUrl}>
```

Comment:
keep the deleted code. Just add on the ccontainer Logo
```javascript
onClick={()=>{
    history.push(/auth/home);
}}

```

Reply:
Ok